artemis-protocols/artemis-openwire-protocol/src/main/java/org/apache/activemq/artemis/core/protocol/openwire/OpenWireMessageConverter.java

```diff
@@ -963,13 +965,14 @@ private static void setAMQMsgObjectProperties(final ActiveMQMessage amqMsg,
          try {
             if (prop instanceof SimpleString) {
                amqMsg.setObjectProperty(keyStr, prop.toString());
+            } else if (prop instanceof byte[]) {
+               ActiveMQServerLogger.LOGGER.propertyConvertedToString(keyStr, "byte[]");
```

Comment:
I'm not sure you should really log a warn for this. This looks regular business to me. it should just to it.